README.md

```diff
@@ -1,5 +1,8 @@
 # The Algorithms - Go
 
+[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.me/TheAlgorithms/100) &nbsp;
```

Comment:
We use [**Liberapay**](https://liberapay.com/TheAlgorithms) now. You may want to change it. 🙂
```suggestion
[![Income](https://img.shields.io/liberapay/receives/TheAlgorithms.svg?logo=liberapay)](https://liberapay.com/TheAlgorithms) &nbsp;
```